### PR TITLE
Update SPOContentSecurityPolicy cmdlets

### PR DESCRIPTION
**Pull request by jaobrie**
Simplify API by removing references to directive.
Unify terminology by preferring "source" over "url".
Add additional tenant flag for early opt-in to enforcement.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOContentSecurityPolicy.md [hunk 1]
```diff
@@ -15,28 +15,27 @@ ms.reviewer:
...
 ```
 
 ## DESCRIPTION
 
-Adds an entry to the **Content Security Policy** configuration. 
-The url in each entry will be added to the corresponding directive during construction of the Content-Security-Policy header.
-In multi-geo environments **Content Security Policy** entries are unique to each geo.
-Entries with a "*" directive will be applied to all directives.
+Adds a source to the **Content Security Policy** configuration. 
+The source will be added to the `script-src` directive during construction of the `Content-Security-Policy` header.
```
Comment (lucabandMSFT) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOContentSecurityPolicy.md:31]: Should we add a pointer to the CSP documentation for what is supported in the source parameter? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/Sources#sources

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentSecurityPolicy.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ Get-SPOContentSecurityPolicy
 
 ## DESCRIPTION
 
-Returns all entries in the current **Content Security Policy** configuration.
-The url in each entry will be added to the corresponding directive during construction of the `Content-Security-Policy` header.
+Returns all sources in the current **Content Security Policy** configuration.
+Each source will be added to the `script-src` directive during construction of the `Content-Security-Policy` header.
```
Comment (lucabandMSFT) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentSecurityPolicy.md:31]: I think this whole paragraph needs to be removed from here.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -3013,7 +3014,23 @@ Accept wildcard characters: False
 When set to `True`, forces a sync of **Content Security Policy** entries for SharePoint framework component in the tenant application catalog.
```
Comment (lucabandMSFT) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3014]: this should say "for SharePoint Framework components". also, you are still using entries here. Wanna change it to sources too?

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -3013,7 +3014,23 @@ Accept wildcard characters: False
...
 New entries will be added to the configuration, if not already present, based on the `cdnBasedPath` property under a solution's `.config/write-manifests.json` if present.
```
Comment (lucabandMSFT) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3015]: entries --> sources

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 5]
```diff
@@ -3013,7 +3014,23 @@ Accept wildcard characters: False
...
-In multi-geo environments, **Content Security Policy** entries are unique to each geo.
+In multi-geo environments, **Content Security Policy** configuration is unique to each geo.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Applicable: SharePoint Online
+Required: False
+Position: Named
+Default value: False
```
Comment (lucabandMSFT) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3025]: I think default value will be true at one point, right? So, ok to start with false but we should remember to change this to "default == true" when we start enforcing CSP

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 6]
```diff
@@ -3013,7 +3014,23 @@ Accept wildcard characters: False
...
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnforceContentSecurityPolicyConfiguration
+
+When set to `True`, opts-in to enforcement of the current **Content Security Policy** configuration.
```
Comment (lucabandMSFT) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3032]: I would change with: "When set to 'True' Content Security Policy violations will be enforced.

1. When set to 'False' violations are not enforced; administrators will find all the Content Security Policy Violations in Microsoft Pureview Portal."

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 5 above)
Comment (jaobrie) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3025]: Currently the way this is implemented is the enforcement flight takes precedence over the flag set here. We could potentially make it so after the flight rolls out the flag is set to true in a timer job but I don't think we want to allow setting it back to false afterwards.

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOContentSecurityPolicy.md (hunk 1 above)
Comment (jaobrie) [reply sharepoint/sharepoint-ps/sharepoint-online/Add-SPOContentSecurityPolicy.md:31]: There's no precedent I can see for linking to the mdn docs, but I do think it's a good idea. Will get Whitney's thoughts on if it's appropriate.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 6 above)
Comment (jaobrie) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3032]: Should we mention the Purview portal before that is ready? We could probably update the docs here with more specific info after the audit events are exposed.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 5 above)
Comment (lucabandMSFT) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3025]: No, actually I think we want to give admins the ability to switch back to report only perpetually. Power Platform folks allow that for example.

**Pull request merged**